src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -34,8 +53,9 @@ on_exit() {
 main() {
     set -f
 
-    [[ $debug ]] && set -x && env | sort
     [[ $debug ]] && export NXF_DEBUG=2
+    [[ $debug ]] && TRACE_CMD="-trace nextflow.plugin"
```

Comment:
This seems to run the debug commands even if debug=false

Could we change this to 
    if $debug ; then
      TRACE_CMD="-trace nextflow.plugin"
      set -x && env | sort
      export NXF_DEBUG=2
    fi

Reply:
Fixed. Thanks!